src/edu/ucsb/cs56/projects/games/fish_animation/Menu.java

```diff
@@ -92,7 +92,7 @@ public void makegui() {
 			PauseBGM.setIcon(new ImageIcon(imgPause));
 		} catch (IOException ex) {
 		}
-
+		
```

Comment:
This change shouldn't be part of this PR.

Reply:
No, it shouldn't. I think I just accidently created new line? I deleted it again. I do not think it will affect the other codes.

Reply:
Sounds good.  You're right that it won't cause any problems.  I just like to keep the Git history clean.  If you see random files changed as part of PRs that should be unrelated you get confused.

Reply:
so do I need to do anything before you merge this PR

Reply:
Yeah.  You need to get rid of this change.

Reply:
fixed